modules/opensearch/src/OpenSearchTingSearchResult.php

```diff
@@ -54,4 +68,27 @@ public function getNumTotalCollections() {
   public function getTingEntityCollections() {
     return $this->openSearchResult->collections;
   }
+
+  /**
+   * Indicates whether the the search could yield more results.
+   *
+   * Eg. by increasing the count or page-number.
+   *
+   * @return bool
+   *   TRUE if the search-provider could provide more results.
+   */
+  public function hasMoreResults() {
+    // The raw value is 0/1 so cast.
+    return (bool) $this->openSearchResult->more;
```

Comment:
Looking back at it I wonder how `hasMoreResults()` relates to results per page, page number, `getNumTotalObjects()`, `getNumTotalCollections()`. I would think one should be able to derive one from the others.

If you can tell why that would not be the case for OpenSearch then it would be nice to add this as a comment.

Reply:
Well, the search-result has a "more" property, so we might as well use it.

But I agree it should basically be the same as (roughly)
($pagesize * ($pagenumber-1) + $results_on_this_page) < getNumTotalObjects()

... but with the number of potential mis-assumptions above I'll happily go for the "more" property :)